pkg/booleanpolicy/validate.go

```diff
@@ -58,6 +58,15 @@ func Validate(p *storage.Policy, options ...ValidateOption) error {
 	for _, section := range p.GetPolicySections() {
 		errorList.AddError(validatePolicySection(section, configuration, p.GetEventSource()))
 	}
+
+	for _, ps := range p.PolicySections {
+		for _, pg := range ps.PolicyGroups {
+			if pg.FieldName == fieldnames.ImageSignatureVerifiedBy && pg.BooleanOperator == storage.BooleanOperator_AND {
```

Comment:
I think that section needs an explaining comment since usually we forbid OR and AND operations. We don't want anyone mistakenly to follow our lead